### ISSUE TITLE
Deletion of multiple entries

### ISSUE THREAD

**Issue by stovocor** [codinguser/gnucash-android#429]
Previous versions of this app allowed the deletion of multiple entries. Please add at least a "clear all" method.

**Comment by codinguser** [comment]
Agreed, a clear all button in the transactions view would not be a bad idea
